UnitySetup/UnitySetup.psm1

```diff
@@ -1310,7 +1313,7 @@ function Start-UnityEditor {
         [parameter(Mandatory = $false)]
         [string]$LogFile,
         [parameter(Mandatory = $false)]
-        [ValidateSet('StandaloneOSX', 'StandaloneWindows', 'iOS', 'Android', 'StandaloneLinux', 'StandaloneWindows64', 'WebGL', 'WSAPlayer', 'StandaloneLinux64', 'StandaloneLinuxUniversal', 'Tizen', 'PSP2', 'PS4', 'XBoxOne', 'N3DS', 'WiiU', 'tvOS', 'Switch')]
+        [ValidateSet('StandaloneOSX', 'StandaloneWindows', 'iOS', 'Android', 'StandaloneLinux', 'StandaloneWindows64', 'WebGL', 'WSAPlayer', 'StandaloneLinux64', 'StandaloneLinuxUniversal', 'Tizen', 'PSP2', 'PS4', 'XBoxOne', 'N3DS', 'WiiU', 'tvOS', 'Switch', 'Lumin')]
```

Comment:
I haven't tried to build for Lumin, but I don't see this as a build target in either of these documentation pages:

https://docs.unity3d.com/Manual/CommandLineArguments.html
https://docs.unity3d.com/ScriptReference/BuildTarget.html

I'm wondering if you've verified the effects of calling `Start-UnityEditor -BuildTarget Lumin`.

Reply:
Unknown. I need to test this out still 
https://forum.unity.com/threads/magic-leap-command-line-usage.560836/#post-4451941

Reply:
Ok. Give it a shot, if it doesn't work, lets drop it from this list until it's supported by Unity. =)

Reply:
Sounds good :)